programmers/실패율.js

```diff
@@ -0,0 +1,22 @@
+function solution(N, stages) {
+   const answers = [];
+   
+   let totalNum = stages.length;
+   let count = 0;
+   let failRate = 0;
+    
+   for (let i = 1; i < N+1; i++) {
+       count = stages.filter(el => i === el).length;
+       if (count === 0) failRate = 0;
+       else  failRate = count / totalNum;
+       totalNum -=  count;
+       answers.push({num: i, ratio:failRate});
+   }
```

Comment:
잘 푸셨네요 !!
중간중간 띄어쓰기가 두 번씩 들어가있거나 들어가지 않은부분이 있어요 :(
컨벤션만 신경써주시면 좋을거 같습니당
```suggestion
   for (let i = 1; i < N + 1; i++) {
       count = stages.filter(el => i === el).length;
       if (count === 0) failRate = 0;
       else failRate = count / totalNum;
       totalNum -= count;
       answers.push({num: i, ratio: failRate});
   }
```